app-drawer/app-drawer.html

```diff
@@ -593,17 +588,20 @@
         if (focusableElements.length > 0) {
           this._firstTabStop = focusableElements[0];
           this._lastTabStop = focusableElements[focusableElements.length - 1];
-          this._firstTabStop.focus();
         } else {
           // Reset saved tab stops when there are no focusable elements in the drawer.
           this._firstTabStop = null;
           this._lastTabStop = null;
         }
-      },
 
-      _removeKeyboardFocusTrap: function() {
-        if (!this.noFocusTrap && this._lastFocusedElement) {
-          this._lastFocusedElement.focus();
+        // Focus on app-drawer if it has non-zero tabindex. Otherwise, focus the first focusable
+        // element in the drawer, if it exists. Use the tabindex attribute since the this.tabIndex
+        // property in IE/Edge returns 0 (instead of -1) when the attribute is not set.
+        var tabindex = this.getAttribute('tabindex');
```

Comment:
In IE/Edge, no tabindex attribute and `tabindex="0"` both make the `tabIndex` property `0`, so there's no way of distinguishing between the two (which we need).